portality/formcontext/formcontext.py

```diff
@@ -492,7 +492,11 @@ def _send_application_approved_email(self, journal_title, publisher_name, email,
 
         to = [email]
         fro = app.config.get('SYSTEM_EMAIL_FROM', 'feedback@doaj.org')
-        subject = app.config.get("SERVICE_NAME", "") + " - journal accepted"
+        if update_request:
```

Comment:
We don't use formcontext.py any more, it will be removed - changes here aren't needed.